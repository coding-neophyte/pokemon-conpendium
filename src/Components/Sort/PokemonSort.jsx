import React from 'react'

export default function PokemonSort({ setSortPokemon, sortPokemon }) {
    return (
        <div>
            <select onChange={(e) => setSortPokemon(e.target.value)}>
                <option value='asc'>ASC</option>
                <option value='desc'>DSC</option>
            </select>
        </div>
    )
}
