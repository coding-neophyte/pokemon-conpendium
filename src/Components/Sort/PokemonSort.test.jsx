import { screen, render } from '@testing-library/react';
import PokemonSort from './PokemonSort.jsx'

it('test pokemon sort', () => {
  const {container} = render(<PokemonSort />);
  expect(container).toMatchSnapshot();
});
