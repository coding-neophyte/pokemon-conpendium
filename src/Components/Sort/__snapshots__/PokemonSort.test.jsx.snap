// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test pokemon sort 1`] = `
<div>
  <div>
    <select>
      <option
        value="asc"
      >
        ASC
      </option>
      <option
        value="desc"
      >
        DSC
      </option>
    </select>
  </div>
</div>
`;
